src/catapult/catapult.mli

```diff
@@ -25,7 +43,7 @@ val on_process_start : t -> program:string -> args:string list -> event
 val on_process_end : t -> event -> unit
 
 (** Emit a counter event. This is measuring the value of an integer variable. *)
```

Comment:
Why do we call this `emit_counter` if it's not even a number in general?
Should it be called `emit_event`, or something like that?

Reply:
This is terminology from the trace format. Counter events (type "C") can have several series in them. There are other kinds of events (not counters) if we wanted to annotate the trace with phases for example ("evaluating rules", "executing rules", etc)